docs/send-data/kubernetes/troubleshoot-collection.md

```diff
@@ -617,3 +974,67 @@ It means that Custom Resource Definition has not been applied by Helm. It is [He
 ```shell
 kubectl apply -f https://raw.githubusercontent.com/open-telemetry/opentelemetry-helm-charts/opentelemetry-operator-0.44.0/charts/opentelemetry-operator/crds/crd-opentelemetry.io_opampbridges.yaml
 ```
+
+## Using Sumo Logic Mock
+
+Sumo Logic Mock is debugging tool, which helps to see what exactly is being send from the Sumo Logic Collection to Sumo Logic. It may help with finding if there are any missing metrics, logs, traces or parts of them like labels or metadata.
```

Comment:
```suggestion
Sumo Logic Mock is debugging tool, which helps to see what exactly is being sent from the Sumo Logic Collection to Sumo Logic. It may help with finding if there are any missing metrics, logs, traces, or parts of them like labels or metadata.
```